content/TrueCommand/Administration/Settings/SAML.md

```diff
@@ -193,116 +190,111 @@ This procedure assumes the Windows administrator user is QE. Substitute your sys
 
    i. Click **Finish**, **Apply**, and **OK**.
 
-8. Close Active Directory.
-
-9. Go to the TrueCommand login page and use the **SAML Login**.
+8. Close **Active Directory**.
 
+9. Go to the TrueCommand login page and use the SAML Login.
 {{< /tab >}}
 {{< tab "Google Admin" >}}
-
-
 To configure Google Admin as the IdP, you must:
 
- • Create a new App for SAML 
- • Configure the SAML app properties to act as the IdP service. 
- • Add the TrueCommand IP and port number as the ACS URL.
- • Configure the SAML app LDAP attributes properties
+* Create a new App for SAML 
+* Configure the SAML app properties to act as the IdP service. 
+* Add the TrueCommand IP and port number as the ACS URL.
+* Configure the SAML app LDAP attributes properties
 
-These top-level steps are described in detail in the example procedure provided below.
+The example procedure below describes these top-level steps in detail. 
 
 ## Activating TrueCommand SAML Service for Google Admin
 
-After configuring SAML in Google Admin, log into your TrueCommand system (i.e., server, container, VM). Go to Config > Administration and then click on the Configuration tab. Scroll down to the SAML settings section.
+After configuring SAML in Google Admin, log into your TrueCommand system (i.e., server, container, VM). Go to **Config > Administration**, then click the **Configuration** tab. Scroll down to the **SAML settings** section.
 
-Enter the XML metadata  file from Google Admin in the  SAML Identity Provider Metadata XML Upload field. After uploading, click Save.
+Enter the XML metadata file from Google Admin into the **SAML Identity Provider Metadata XML Upload** field, then click **Save**.
 
-Click Start the SAML service checkbox and click Save to start the service.
+Click the **Start the SAML service** checkbox, then click **Save** to start the service.
 
 Log out of TrueCommand.
 
 ## Configuring a Google Admin SAML App
 
-1. Open Google Admin and go to Apps > Web and mobile apps
+1. Open Google Admin and go to **Apps > Web and mobile apps**
 
    ![GASelectWebandMobileApps](/images/SAML/GASelectWebandMobileApps.jpg "GA Select Web and Mobile Apps")
     
-2. Click Add App and select Add custom SAML app.The Add custom SAML app screen App details screen displays. 
+2. Click **Add App**, then select **Add custom SAML app**.
```

Comment:
The original sentence should change as follows (or can remove the screen, and can change to a >):

2. Click **Add App** and select **Add custom SAML app**. The **Add custom SAML app** screen, **App details** screen displays.

Reply:
Changed it to: 
Click **Add App**, then select **Add custom SAML app** to open the **App details** screen.

Reply:
Excellent!